### ISSUE TITLE
A white screen is displayed and the video is not visible.

### ISSUE THREAD

**Issue by geumu** [montoyo/webdisplays#102]
Minecraft Version: 1.12.2
Forge Version: 1.12.2 - 14.23.5.2854
Mod Version:
  - mcef-1.12.2-1.11.jar
  - webdisplays-1.12.2-1.1.jar
Explanation: A white screen is displayed and the video is not visible.
Video: https://youtu.be/zfGXp3eRpIE

**Comment by BigLDP** [comment]
Same issue, I believe it is MCEF's issue.

**Comment by geumu** [comment]
> Same issue, I believe it is MCEF's issue. The minepads also don't work

Is it caused by the problem of not getting mcef2.json from the developer site?

**Comment by BigLDP** [comment]
> > Same issue, I believe it is MCEF's issue. The minepads also don't work
> 
> Is it caused by the problem of not getting mcef2.json from the developer site?

Yeah, I evidently do not have it, but that would be created by running mcef most likely with mc as well as the config.

**Comment by ItsK1tty** [comment]
what are your java versions, people? try updating it

**Comment by BigLDP** [comment]
> what are your java versions, people? try updating it

latest of everything, latest mods, latest mcef, latest minecraft launcher and java.

**Comment by ItsK1tty** [comment]
> **Minecraft Version**: 1.12.2
> **Forge Version**: 1.12.2 - 14.23.5.2854
> **Java Version**: jdk1.8.0_271
> **Mod Version**:
>   - mcef-1.12.2-1.11.jar
>   - webdisplays-1.12.2-1.1.jar
> 
> **Explanation**: A white screen is displayed and the video is not visible.
> **Video**: https://youtu.be/zfGXp3eRpIE
> **Log**: https://pastebin.com/k2BW1292
> 
> I have read this article.
> https://github.com/montoyo/webdisplays/issues/15
> When I accessed the site to follow the instructions to allow access to http://montoyo.net, it was currently under maintenance.
> Does my problem have anything to do with the'http://montoyo.net' site being maintained?
> 

your java is u271, however Minecraft tries to use u51. consider dealing with it

**Comment by ItsK1tty** [comment]
> > what are your java versions, people? try updating it
> 
> latest of everything, latest mods, latest mcef, latest minecraft launcher and java 64 bit with 8gb allocated.

decrease it to 4gb and wipe your all MCEF related files (or just wipe your Minecraft folder). if that doesn't help, get 1.0 WD and 0.9 MCEF

**Comment by montoyo** [comment]
> Does my problem have anything to do with the'http://montoyo.net' site being maintained?
montoyo.net is not really under maintenance :p it's just me not having the time to create a proper homepage.
As long as https://montoyo.net/jcef/ is online, there shouldn't be any problem.

I believe the issue is just that Let's Encrypt SSL certificate embedded in MCEF is out-of-date. I'll have to regenerate a binary. In the meantime, you could try to set 'secureMirrorsOnly' to false in the config file.

**Comment by ItsK1tty** [comment]
montoyo! long time no see, we still hope you can bring it back to life, cause people start to use it more and more (as there are more and more issues open). can we still hope for a great future?

**Comment by montoyo** [comment]
> montoyo! long time no see, we still hope you can bring WD back to life, cause people start to use it more and more (as there are more and more issues open). can we still hope for a great future?
Yeah not really... am wayyy to busy IRL... Looking for someone to take over both MCEF and WD

However I believe I fixed the HTTPS bug here with MCEF; @geumu @BigLDP try to download MCEF 1.11b

**Comment by NFSMW2005** [comment]
I am really confused I tried but I can't find my files I am really confused if anyone has discord can we call so I could get help?

**Comment by ItsK1tty** [comment]
what files are you talking about?

**Comment by NFSMW2005** [comment]
> what files are you talking about?

Mcef2.json and the latest log files

**Comment by ItsK1tty** [comment]
you can find mcef2.json using standard windows search in the file explorer, and logs are in logs/ folder in your Minecraft folder

**Comment by NFSMW2005** [comment]
> you can find mcef2.json using standard windows search in the file explorer, and logs are in logs/ folder in your Minecraft folder

It is not there I looked for 2 hours

**Comment by ItsK1tty** [comment]
are you sure you're in the folder your Minecraft is installed? that folder should have at least logs, config, mods, versions and crash-reports folders inside

**Comment by NFSMW2005** [comment]
> are you sure you're in the folder your Minecraft is installed? that folder should have at least logs, config, mods, versions and crash-reports folders inside

Yes I am in the correct folder

**Comment by ItsK1tty** [comment]
then, navigate into the `logs` folder and send `latest.log` file

**Comment by NFSMW2005** [comment]
[2021-01-18-6.log.gz](https://github.com/montoyo/webdisplays/files/5837300/2021-01-18-6.log.gz) there you go

**Comment by NFSMW2005** [comment]
> then, navigate into the `logs` folder and send `latest.log` file

[latest.log](https://github.com/montoyo/webdisplays/files/5837412/latest.log) here you go

**Comment by ItsK1tty** [comment]
sigh, there's no even sense of mcef or anything connected to webdisplays but just logs of you playing on hypixel. do you have multiple profiles in your minecraft launcher?

**Comment by NFSMW2005** [comment]
> sigh, there's no even sense of mcef or anything connected to webdisplays but just logs of you playing on hypixel. do you have multiple profiles in your minecraft launcher?

No I don’t it is just my account do you want me to bring up the latest log or do you want me to try and send it again

**Comment by ItsK1tty** [comment]
nothing will change if you will send it again unless you run it with webdisplays on

**Comment by xPopbobx** [comment]
i have the same problem, cant just someone send mcef2.json?

**Comment by ReleasedBlock33YT** [comment]
I have the same issue, and I can't find mcef2.json nor fml-client-latest.log. _Does anyone_ have either of those two files??

**Comment by geumu** [comment]
i dont have this ;-;

**Comment by FryDaDog** [comment]
i need help too
